zingolib/src/wallet/transactions.rs

```diff
@@ -1,7 +1,55 @@
 use std::collections::HashMap;
 
-use crate::wallet::{data::WitnessTrees, transaction_records_by_id::TransactionRecordsById};
+use zcash_client_backend::PoolType;
+use zcash_primitives::transaction::TxId;
 
+use crate::wallet::{
+    data::WitnessTrees, notes::NoteRecordIdentifier, traits::DomainWalletExt,
+    transaction_records_by_id::TransactionRecordsById,
+};
+
+use super::data::TransactionRecord;
+
+#[derive(Debug)]
```

Comment:
This seems to be obsolete the type is named `TransactionRecordsById` in dev, and has moved.